datasource/opennms-direct/src/main/java/org/opennms/oce/datasource/opennms/jvm/Mappers.java

```diff
@@ -51,19 +48,31 @@
 import org.opennms.oce.datasource.common.ImmutableAlarm;
 import org.opennms.oce.datasource.common.ImmutableAlarmFeedback;
 import org.opennms.oce.datasource.common.ImmutableSituation;
-import org.opennms.oce.datasource.common.inventory.ManagedObjectType;
+import org.opennms.oce.datasource.common.ScriptedInventoryException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.google.common.base.Enums;
-import com.google.common.base.Strings;
 
 public class Mappers {
     private static final Logger LOG = LoggerFactory.getLogger(Mappers.class);
     public static final String SITUATION_UEI = "uei.opennms.org/alarms/situation";
     public static final String SITUATION_ID_PARM_NAME = "situationId";
 
-    public static Alarm toAlarm(org.opennms.integration.api.v1.model.Alarm alarm) {
+    private final ScriptedInventoryService inventoryService;
+
+    /**
+     * @param inventoryService wires in the JSR 223 script
+     */
+    public Mappers(ScriptedInventoryService inventoryService) {
+        this.inventoryService = inventoryService;
+    }
+
+    public void init() {
+        // required for karaf.
```

Comment:
This is only required since `init-method="init"` is set in the Blueprint - if there's no initialization to be done, then this can be removed.

Reply:
cleaned up